tests/client/net_handler_client.c

```diff
@@ -425,7 +425,6 @@ void send_gamepad_state(uint32_t buttons, float joystick_vals[4]) {
     // free everything
     free(gp_state.axes);
     free(send_buf);
-    usleep(400000);  // allow time for net handler and runtime to react and generate output before returning to client
```

Comment:
this might break some other test stuff, not sure though.

Reply:
It did, but it's necessary for how Daniel's keyboard_interface is implemented (We're added appropriate fixes in some test cases)